File: Python/car.py
from account import Account
class car:
    id = int
    license = str
    driver = Account("","")
    passenger = int

    def __init__(self, license, driver):# COnstructor to initialize the car
        self.license = license
        self.driver = driver

    def printall (self):
        print(str(self.id) + " " + self.license + " "+ self.driver.name + " "+str(self.passenger) )

File: JS/Car.js
function Car(license, driver){
    //here you add the parameters license and driver 
    //to invoke th constructor method and obligue the 
    //usage of the mentioned parameters
    this.id;
    this.license=license; // in this line, we assign to the class the parameters that are received when creating the class.
    this.driver=driver;
    this.passenger;
}
//the below function will be a method from the class
// as seen below is the way to call them. 
Car.prototype.Printall=function(){
    console.log(this.driver)
    console.log(this.driver.name)
    console.log(this.driver.document)
}